### DISCUSSION TITLE
when can we expect a new version?

### DISCUSSION THREAD

**Discussion by Becker884** [grafana/grafana#63390] [General]
I´m waiting since end of january for a new verison, I´m using the buggy 9.3.6 and I don´t want to change to beta channel.
usually a new version comes out every 2-10 days ?
Greetings

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
